src/components.js

```diff
@@ -1211,6 +1211,19 @@ const components = [
     },
     style: {}
   },
+
+  {
+    name: 'Flex',
+    type: Flex,
+    props: {},
+    style: {}
+  },
```

Comment:
The components array here goes through the custom Rebass component factory, which would wrap the Flex and Box components from grid-styled, but I'd rather pull those in directly from the grid-styled package to handle the underlying div in that library – the trick to not passing style props down to a styled-components' HTML element needs to live in the initial styled-component definition due to the way their higher order component works